### PR TITLE
Upgrade to Spring Boot 3.2 and Spring Cloud 2023

### PR DESCRIPTION
**Pull request by danielthirion**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: k8s/overlays/main_versions/test/kustomization.yaml [hunk 1]
```diff
@@ -39,79 +39,79 @@ images:
...
-    newTag: v1.10.0
+    newTag: v1.11.0
   - name: gridcapa-swe-rao-logs-dispatcher
     newName: docker.io/farao/gridcapa-rao-logs-dispatcher
-    newTag: v1.3.1
+    newTag: v1.4.0
   - name: gridcapa-swe-runner
     newName: docker.io/farao/gridcapa-swe-runner
-    newTag: v1.17.1.1
+    newTag: v1.17.3
```
Comment (vbochetRTE) [inline k8s/overlays/main_versions/test/kustomization.yaml:90]: This version of swe-runner is based on the buggy powsybl-core v6.2.4 dependency. Wait for the next version before validating this PR.

File: docker-compose/cse-import-ec-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml [hunk 2]
```diff
@@ -5,6 +5,10 @@ spring:
     username: cse_import_ec_d2cc_server
     password: cse-import-ec-d2cc
   jpa:
+    properties:
```
Comment (vbochetRTE) [inline docker-compose/cse-import-ec-d2cc/config/cse-d2cc-gridcapa-task-manager-application.yml:8]: Why didn't you add the same changes to CSE Import EC IDCC, CSE Import IDCC and SWE IDCC IDCF?

File: docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml [hunk 3]
```diff
@@ -9,6 +9,8 @@ spring:
     template:
       reply-timeout: 7200000
   cloud:
+    function:
```
Comment (vbochetRTE) [inline docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml:12]: Why did you add this here, and not in any other process-runner-configuration ?

File: docker-compose/swe-d2cc/config/data.sql [hunk 4]
```diff
@@ -1,109 +1,109 @@
-INSERT INTO parameter (id, name, display_order, parameter_type, section_title, section_order, value)
```
Comment (vbochetRTE) [inline docker-compose/swe-d2cc/config/data.sql:1]: Why don't you change this for SWE IDCC IDCF ?

File: k8s/bases/cse-valid-d2cc/cse-valid-d2cc-task-manager-configmap.yaml [hunk 5]
```diff
@@ -22,6 +22,10 @@ data:
           maximumPoolSize: ${HIKARI_MAX_POOL_SIZE:10}
           minimumIdle: ${HIKARI_MIN_IDLE:5}
       jpa:
+        properties:
```
Comment (vbochetRTE) [inline k8s/bases/cse-valid-d2cc/cse-valid-d2cc-task-manager-configmap.yaml:25]: Why didn't you perform changes for CSE Valid processes in docker-compose folder?

File: docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml (hunk 3 above)
Comment (danielthirion) [reply docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml:12]: You're right, that's a mistake that happened while i was experimenting changes required. It should have been deleted afterwards

**Review by vbochetRTE — APPROVED** [review]
